plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/MySqlQueryRunner.java

```diff
@@ -76,6 +81,18 @@ public static DistributedQueryRunner createMySqlQueryRunner(
         }
     }
 
+    private static void provisionTables(QueryRunner queryRunner, Iterable<TpchTable<?>> tables)
+    {
+        Session session = createSession();
+        Set<String> existingTables = queryRunner.listTables(session, "mysql", TPCH_SCHEMA).stream()
+                .map(QualifiedObjectName::getObjectName)
+                .collect(toImmutableSet());
+
+        Streams.stream(tables)
+                .filter(table -> !existingTables.contains(table.getTableName().toLowerCase(ENGLISH)))
+                .forEach(table -> copyTable(queryRunner, "tpch", TINY_SCHEMA_NAME, table.getTableName().toLowerCase(ENGLISH), session));
```

Comment:
Maybe we just add a param to "create if not exists" to `copyTpchTables`  method, or introduce a similar reusable method for this? The code being added here is pretty repetitive, and yet not 100% mechanical (eg lowercasing)